### ISSUE TITLE
Checking pandoc-citeproc info failes

### ISSUE THREAD

**Issue by lasest** [manubot/manubot#286]
According to pandoc-citeproc repository (https://github.com/jgm/pandoc-citeproc) the package is no longer maintained. Last update was pushed 6 months ago.

Pandoc-citeproc depends on haskell libraries (for example, libHSyaml). Those libraries receive very frequent updates. For those, who use rolling-release linux distributions this creates problems. For example, the latest version of pandoc-citeproc won't work with the latest versions of haskell packages on Arch Linux. Due to the failure of pandoc-citeproc, `manubot cite` also wouldn't work.

`Manubot cite` command attempts to start pandoc-citeproc in `get_pandoc_info()` (pandoc/util.py). However, pandoc-citeproc is only required for older pandoc versions.

It would make more sense to implement the following logic in get_pandoc_info():

1. First get info about pandoc
2. If pandoc version is lower than 2.11, only then get info about pandoc citeproc

**Comment by dhimmel** [comment]
Thanks for https://github.com/manubot/manubot/pull/287 and this report.

> For those, who use rolling-release linux distributions this creates problems. For example, the latest version of pandoc-citeproc won't work with the latest versions of haskell packages on Arch Linux.

Are you saying that running `pandoc-citeproc --version` on your system fails?

> First get info about pandoc. If pandoc version is lower than 2.11, only then get info about pandoc citeproc

I agree this makes sense. There are some features that pandoc-citeproc supports that are not part of pandoc 2.11+, such as the conversion of uncommon bibliography formats. However, since pandoc-citeproc is no longer maintained, I don't think it's reasonable for manubot to continue supporting these formats either. As far as we know, no one is using these feature.

**Comment by lasest** [comment]
> Are you saying that running pandoc-citeproc --version on your system fails?

Yes, it appears that Arch Linux removed pandoc-citeproc from their repositories. The rolling release version of OpenSuse also doesn't provide it. However both distributions do have this package: https://github.com/jgm/citeproc

Compiling pandoc-citeproc from sources solves the issue, but it takes a while and I don't think it's good to require users to do it.

I'm concerned that other distributions might stop supporting the package in the future, although both 2021.04 version of Ubuntu and OpenSuse Leap 15.2 still have it.

**Comment by dhimmel** [comment]
@lasest do you still happen to have the error message handy that you got when `pandoc-citeproc --version` failed? I'm thinking that adding it here will help other users search for this issue.

#267 is merged and released in https://pypi.org/project/manubot/0.5.1/. Did you encounter this error when tying to use a rootstock-derived manuscript or when using the `manubot` package directly? We still need to update rootstock.

**Comment by lasest** [comment]
I don't have the exact error message and I can't reproduce it since I removed the old pandoc-citeproc package which I used to have. The message was something like:

`error while loading shared libraries: libHSyaml-0.11.5.0-4joGGT1xFvc3jg9Hr1jCly-ghc8.10.4.so: cannot open shared object file: No such file or directory` (the filename was exactly so).

I tried creating a symlink to the newer version of that library which I have, but then it would ask for another haskell library, so I didn't try to find out all of them.

I used `manubot cite` directly.
